www/src/examples/Accordion/ContexAwareToggle.js

```diff
@@ -0,0 +1,45 @@
+function ContexAwareToggle({ children, eventKey, callback }) {
```

Comment:
**nitpick:** should be `ContextAwareToggle`

Reply:
good catch, I copied and pasted that around without noticing it. fixed.